development/tools/jobs/kyma/kyma_compass_provisioner_tests_test.go

```diff
@@ -1,95 +0,0 @@
-		preset.KymaKeyring,
-		preset.KymaEncriptionKey,
-		"preset-kms-gc-project-env",
-		preset.SaGKEKymaIntegration,
-		"preset-gc-compute-envs",
-		preset.GCProjectEnv,
-		"preset-docker-push-repository-gke-integration",
-		preset.DindEnabled,
-		"preset-kyma-artifacts-bucket",
-		preset.GardenerAzureIntegration,
-		preset.BuildPr,
-		preset.ClusterVersion,
-	)
-	tester.AssertThatHasExtraRefTestInfra(t, actualJob.JobBase.UtilityConfig, "master")
-	require.Len(t, actualJob.Spec.Containers, 1)
-	compassCont := actualJob.Spec.Containers[0]
-	assert.Equal(t, tester.ImageKymaIntegrationLatest, compassCont.Image)
-	assert.Equal(t, []string{"bash"}, compassCont.Command)
-	require.Len(t, compassCont.Args, 2)
-	assert.Equal(t, "-c", compassCont.Args[0])
-	assert.Equal(t, "${KYMA_PROJECT_DIR}/test-infra/prow/scripts/cluster-integration/kyma-gke-compass-integration.sh", compassCont.Args[1])
-	tester.AssertThatContainerHasEnv(t, compassCont, "CLOUDSDK_COMPUTE_ZONE", "europe-west4-b")
-	tester.AssertThatContainerHasEnv(t, compassCont, "RUN_PROVISIONER_TESTS", "true")
-	tester.AssertThatSpecifiesResourceRequests(t, actualJob.JobBase)
-}
-
-func TestKymaGKECompassProvisionerTestsPostsubmit(t *testing.T) {
-	// given
-	jobConfig, err := tester.ReadJobConfig("./../../../../prow/jobs/kyma/kyma-compass-provisioner-tests.yaml")
-	require.NoError(t, err)
-
-	// when
-	actualJob := tester.FindPostsubmitJobByNameAndBranch(jobConfig.AllStaticPostsubmits([]string{"kyma-project/kyma"}), "post-master-kyma-gke-compass-provisioner-tests", "master")
-	require.NotNil(t, actualJob)
-
-	// then
-	assert.True(t, actualJob.Decorate)
-	assert.Equal(t, "github.com/kyma-project/kyma", actualJob.PathAlias)
-	assert.Equal(t, 10, actualJob.MaxConcurrency)
-	tester.AssertThatHasPresets(t, actualJob.JobBase,
-		preset.KymaGuardBotGithubToken,
-		preset.KymaKeyring,
-		preset.KymaEncriptionKey,
-		"preset-kms-gc-project-env",
-		preset.SaGKEKymaIntegration,
-		"preset-gc-compute-envs",
-		preset.GCProjectEnv,
-		"preset-docker-push-repository-gke-integration",
-		preset.DindEnabled,
-		"preset-kyma-artifacts-bucket",
-		preset.GardenerAzureIntegration,
-		preset.BuildMaster,
-		preset.ClusterVersion,
-	)
-	tester.AssertThatHasExtraRefTestInfra(t, actualJob.JobBase.UtilityConfig, "master")
-	require.Len(t, actualJob.Spec.Containers, 1)
-	compassCont := actualJob.Spec.Containers[0]
-	assert.Equal(t, tester.ImageKymaIntegrationLatest, compassCont.Image)
-	assert.Equal(t, []string{"bash"}, compassCont.Command)
-	require.Len(t, compassCont.Args, 2)
-	assert.Equal(t, "-c", compassCont.Args[0])
-	assert.Equal(t, "${KYMA_PROJECT_DIR}/test-infra/prow/scripts/cluster-integration/kyma-gke-compass-integration.sh", compassCont.Args[1])
-	tester.AssertThatContainerHasEnv(t, compassCont, "CLOUDSDK_COMPUTE_ZONE", "europe-west4-b")
-	tester.AssertThatContainerHasEnv(t, compassCont, "RUN_PROVISIONER_TESTS", "true")
-	tester.AssertThatSpecifiesResourceRequests(t, actualJob.JobBase)
-}
```

Comment:
I'm not sure if we want to remove that. I can recall that `provisioner-tests` still have some value for us and maybe w should consider replacing `kyma-gke-compass-integration.sh` or just rename it to `kyma-gke-provisioner-tests.sh` and it will be used only in this pipeline.

Reply:
@kyma-project/framefrog, @franpog859

Reply:
Ok, nvm. All clear now. This is also tested in `control-plane-gke-provisioner-integration`

Reply:
Verified. The mentioned test was pre-master-control-plane-gke-provisioner-integration and it was not removed.